src/overlays/actors/ovl_En_Raf/z_en_raf.c

```diff
@@ -261,33 +252,54 @@ void EnRaf_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
+typedef enum CarnivorousLilyAnimation {
+    /* 0 */ CARNIVOROUS_LILY_ANIM_IDLE,
+    /* 1 */ CARNIVOROUS_LILY_ANIM_CLOSE,
+    /* 2 */ CARNIVOROUS_LILY_ANIM_CHEW,
+    /* 3 */ CARNIVOROUS_LILY_ANIM_SPIT,
+    /* 4 */ CARNIVOROUS_LILY_ANIM_CONVULSE,
+    /* 5 */ CARNIVOROUS_LILY_ANIM_DEATH,
+    /* 6 */ CARNIVOROUS_LILY_ANIM_MAX
+} CarnivorousLilyAnimation;
```

Comment:
You changed the prefix of this from `EN_RAF` to `CARNIVOROUS_LILY`, but all the other enums still start with `EN_RAF`. Can you either:
- Change all other enums to start with `CARNIVOROUS_LILY`, or
- Change this enum back to start with `EN_RAF`?

It's a minor thing, but there's inconsistency here that wasn't there before.